Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ContentGraph.php

```diff
@@ -359,13 +373,6 @@ private function fetchRows(QueryBuilder $queryBuilder): array
         }
     }
 
-    /** The workspace this content graph is operating on */
-    public function getWorkspaceName(): WorkspaceName
-    {
-        return $this->workspaceName;
-    }
-
-    /** @internal The content stream id where the workspace name points to for this instance */
```

Comment:
Why should this not be internal?

Reply:
the whole implementation is internal. Only the interface is api. There is no need to add / duplicate doc comments here imo.